Backend/routes/auth.js

```diff
@@ -0,0 +1,108 @@
+const express = require('express');
+const { query, validationResult, body } = require('express-validator');
+const router = express.Router();
+const User = require('../models/User');
+const bcrypt = require('bcryptjs');
+var jwt = require('jsonwebtoken');
+const fetchuser = require('../middleware/fetchUser');
+
+const JWT_SECRET = "Hridhima$react"
```

Comment:
Kindly use .env file, dont hardcode these secrets

Reply:
got it
thankyou